source/_integrations/nws.markdown

```diff
@@ -25,6 +25,8 @@ According to the [API documentation](https://www.weather.gov/documentation/servi
 
 Providing a METAR station code is optional, and if not supplied, the closest station to the latitude and longitude will be chosen. A list of nearby stations is printed to the log with level `DEBUG` if no station is supplied. Stations can also be found on the [NOAA website](https://www.cnrfc.noaa.gov/metar.php). Codes with only three characters, for example, `ADW` should be prefixed with the letter K, `KADW`.
 
-Two weather entities are created for each entry in the configuration: one for day and night forecasts and one for hourly forecasts. The hourly forecast entity is disabled after configuration but can be enabled by the user. The time supplied for each forecast is the start time for the forecast. Sensors are also created as disabled entities after configuration and can be enabled by the user.
+One weather entity is created for each entry in the configuration for day and night forecasts. The time supplied for each forecast is the start time for the forecast. Hourly forecast data can be obtained using the `weather.get_forecasts` service. Sensors are also created as disabled entities after configuration and can be enabled by the user.
```

Comment:
The way you have worded this makes it sound like the entity is created for day/night forecast and the hourly forecast needs to be obtained via `get_forecasts`.  In reality, one entity is for both forecasts and `get_forecasts` can be used for either type.

Reply:
Should I change "day and night" to "day/night" or do you have another suggestion on how to clarify this?  Perhaps I need to read the latest change info and day/night isn't even accurate anymore, but that's the one entity I got when I installed in October...

Reply:
That verbiage was original verbiage that I didn't touch.  After reviewing the release notes, I have modified that language and added a commit.  I am marking this concern as resolved accordingly.